pkg/ocm/helpers_test.go

```diff
@@ -695,3 +695,90 @@ var _ = Describe("GetAutoNodeRoleArn", func() {
 		Expect(exists).To(BeTrue())
 	})
 })
+
+var _ = Describe("ValidateHTTPProxy", func() {
+	It("accepts empty string", func() {
+		err := ValidateHTTPProxy("")
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("accepts valid http proxy without authentication", func() {
+		err := ValidateHTTPProxy("http://proxy.example.com:8080")
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("accepts valid http proxy with basic authentication", func() {
+		err := ValidateHTTPProxy("http://user:password@proxy.example.com:8080")
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("rejects http proxy with unencoded forward slash in password and provides helpful error", func() {
+		// This is the exact example from the bug report
```

Comment:
Maybe put the ticket number in, when someone looks at this in a year they won't know what `this` nor `the bug report` is